ThunderCloud/TSCAuthenticationController.m

```diff
@@ -18,7 +18,8 @@ - (instancetype)init
 {
     if (self = [super init]) {
         
-        [TSCDeveloperModeController shared];
+        TSCDeveloperModeController *devModeController = [TSCDeveloperModeController shared];
+        devModeController.baseURL = devModeController.baseURL;
```

Comment:
Hmmm I can't go back to a time when that shared call wasn't in the code, I don't see why it would be causing issues though?